README.md

```diff
@@ -5,4 +5,8 @@
 + Create an object
 + Add key-value pairs to an object
 + Replace values in an object
-+ Delete key-value pairs in an object
\ No newline at end of file
++ Delete key-value pairs in an object
+
+## Instructions
+
++ Create a function `addIngredient` that accepts three arguments: a recipe object, an ingredient name (a key in the object) and the amounts (the value of the ingredient key). The body of the function should add the key-value pair to the recipe object, and then return the updated object.
```

Comment:
maybe add an example of how this will be used? 

```js
var tacoIngredients = {tortillas: "1 dozen, small"}
addIngredient(tacoIngredients, "cheese", "a lot")
tacoIngredients //=> {tortillas: "1 dozen, small", cheese: "a lot"}
```

Also, this needs instructions for the other three functions.